docs/best-practices/mocking-request.md

```diff
@@ -5,17 +5,20 @@ Undici have its own mocking [utility](../api/MockAgent.md). It allow us to inter
 Example:
 
 ```js
-// index.mjs
+// bank.mjs
 import { request } from 'undici'
 
 export async function bankTransfer(recepient, ammount) {
-  const { body } = await request('http://localhost:3000/bank-transfer', 
+  const { body } = await request('http://localhost:3000/bank-transfer',
```

Comment:
According to the `undici` [`.editorconfig`](https://github.com/nodejs/undici/blob/main/.editorconfig#L9), trimming trailing whitespace is the preferred style.